aytests/no_restart_second_stage.sh

```diff
@@ -4,4 +4,4 @@ set -e -x
 
 # bug 956043 - YAST second stage must not restarted while activating
 
-grep "Skipping service YaST2-Second-Stage - it's already in desired state" /var/log/YaST2/y2log && echo "AUTOYAST OK"
+grep -E ".*restarting services.*YaST2-Second-Stage" /var/log/YaST2/y2log || echo "AUTOYAST OK"
```

Comment:
There's no trace in the y2log that the Second Stage was not called. So let's keep the test as it is.

Thanks!